src/reports/aggregate/global/factionscombat.entity.ts

```diff
@@ -1,8 +1,10 @@
 import { Column, ObjectIdColumn, Entity, ObjectID } from 'typeorm';
 import { World, worldArray } from '../../../constants/world.consts';
-import FactionCombat from '../common/combatstats.embed';
+import CombatStats from '../common/combatstats.embed';
 
-@Entity()
+@Entity({
+  name: 'aggregate_global_factionscombat'
```

Comment:
factions - needs to be faction to match the pattern of the rest of the entitites, make sure to rename the file as well